.github/workflows/csharp.yaml

```diff
@@ -1,6 +1,6 @@
 name: dotnet package
 
-#on: [pull_request]
+on: [push]

```

Comment:
Why you decide to change it? In other CI builds we use pull_request

Reply:
I did it for testing, but forgot to change it back

Reply:
Get it